api-migration.md

```diff
@@ -0,0 +1,28 @@
+# Migrating to new APIs
+
+This document covers migrating your code when a nodejs-cloudant release has
+breaking API changes. Each section covers migrating from one particular
+version to another. The section titles state the specific versions between
+which the change was made; often one can migrate from earlier versions than
+the one specified on the left of the title to the new version specified
+on the right.
+
+Feel free to file bugs if the instructions don't work for you; we'll use
+these to improve the documentation and we'll obviously also try to help
+you through the process.
+
+## 1.10.0 → 2.0.0
```

Comment:
I think this can change from 1.x to 2.x and tweak the text above to re-assure the user that breaking changes typically (hopefully only) occur on major version changes

Reply:
Will do.